src/ballet/sbpf/fuzz_sbpf_loader.c

```diff
@@ -34,7 +34,7 @@ LLVMFuzzerTestOneInput( uchar const * data,
                         ulong         size ) {
 
   fd_sbpf_elf_info_t info;
-  if( FD_UNLIKELY( !fd_sbpf_elf_peek( &info, data, size, true ) ) )
+  if( FD_UNLIKELY( !fd_sbpf_elf_peek( &info, data, size, 0 ) ) )
```

Comment:
why 0?

Reply:
@ravyu-jump Please see the Slack link above, KB recommends using int/0/1